src/main/java/org/broadinstitute/hellbender/engine/ReadWalker.java

```diff
@@ -21,7 +22,7 @@
  * ReadWalker authors must implement the apply() method to process each read, and may optionally implement
  * onTraversalStart() and/or onTraversalSuccess(). See the PrintReadsWithReference walker for an example.
  */
-public abstract class ReadWalker extends GATKTool {
+public abstract class ReadWalker extends GATKTool implements CommandLinePluginProvider {
```

Comment:
Is there a reason `CommandLineProgram` doesn't implement this?  It seems like it should since it implements `getPluginDescriptors()`?